src/main/java/org/jabref/gui/entryeditor/LatexCitationsTabViewModel.java

```diff
@@ -160,19 +170,41 @@ private List<Path> searchDirectory(Path directory) {
         }
     }
 
+    private void setListener(Path path) {
+        if (!hasListener) {
+            try {
+                fileUpdateMonitor.addListenerForFile(path, this::refreshLatexDirectory);
+                hasListener = true;
+            } catch (IOException e) {
+                LOGGER.error("Could not find file", e);
+            }
+        }
+    }
+
+    private void setListener(Path oldPath, Path newPath) {
+        try {
+            fileUpdateMonitor.removeListener(oldPath, this::refreshLatexDirectory);
+            fileUpdateMonitor.addListenerForFile(newPath, this::refreshLatexDirectory);
+        } catch (IOException e) {
+            LOGGER.error("Could not find file", e);
```

Comment:
```suggestion
            LOGGER.error("Could update listener", e);
```

Reply:
Sorry about that, did not know that "Commit suggestion" was a feature in GitHub